js-intermediate.js

```diff
@@ -14,7 +21,14 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [7, 23, -823]
 
+oddChecker = (arr) => {
+  let newArr = arr.filter(n => typeof n === "number")
+  let newArr2 = newArr.filter(n => n%2 !== 0)
+  return newArr2
+} 
 
+console.log(oddChecker(fullArr1))
+console.log(oddChecker(fullArr2))
 
```

Comment:
You logic is great, you can shorten your code by having the first filter method handle both the typeof and the odd/even